### PR TITLE
Minor improvements to Python scripts

### PR DESCRIPTION
**Pull request by rettinghaus**

### CODE REVIEW

File: doc/test-suite-diff.py [hunk 1]
```diff
@@ -20,7 +21,7 @@ def start_webpage():
     head = etree.Element('head')
     html.append(head)
     title = etree.SubElement(head, 'title')
-    title.text = "Verovio test suite for Pull Request"
+    title.text = 'Verovio test suite for Pull Request'
```
Comment (ahankinson) [inline doc/test-suite-diff.py:24]: Not sure why you’re changing to single quotes?

File: doc/test-suite-diff.py [hunk 2]
```diff
@@ -130,11 +131,11 @@ def end_webpage(html, body, htmlOutFile):
             pngFileOut = os.path.join(path_out, item1, name + '.png')
             pngFile1Out = os.path.join(path_out, item1, name + '.after.png')
             pngFile2Out = os.path.join(path_out, item1, name + '.before.png')
-            print(pngFile1, pngFile2)
+            print(f'Comparing {name}')
```
Comment (ahankinson) [inline doc/test-suite-diff.py:134]: This is actually changing the output of print, so it’s not simply a formatting change

File: fonts/generate_font_metadata.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def validateGlyph(glyph):
             report += '- DUPLICATE UNICODE POINT'
 
         if report != '':
-            print('%s (%#0.2x) issues: %s' % (g.glyphname, glyph.unicode, report), log)
```
Comment (ahankinson) [inline fonts/generate_font_metadata.py:48]: F strings are used elsewhere — why not here?

File: setup.py [hunk 4]
```diff
@@ -12,6 +12,7 @@
 
 
 class build_ext(_build_ext):
+
```
Comment (ahankinson) [inline setup.py:15]: Strange place for an empty line, especially with a docstring following it

**Review by ahankinson — CHANGES_REQUESTED** [review]
Some questions about changes

File: setup.py (hunk 4 above)
Comment (rettinghaus) [reply setup.py:15]: PEP 257

Comment (ahankinson) [reply setup.py:15]: PEP257 doesn't specify a blank line before the docstring, it specifies a blank line after.

File: doc/test-suite-diff.py (hunk 1 above)
Comment (rettinghaus) [reply doc/test-suite-diff.py:24]: Mixing single with double quotes is just bad style. (PEP8)

File: doc/test-suite-diff.py (hunk 2 above)
Comment (rettinghaus) [reply doc/test-suite-diff.py:134]: Correct, but simply printing the same file name twice with full paths isn't very informative, so it's still a kind of formatting change ;-)

File: setup.py (hunk 4 above)
Comment (rettinghaus) [reply setup.py:15]: See Error Code [D203](https://pep257.readthedocs.io/en/latest/error_codes.html) 

> 1 blank line required before class docstring

But we can leave it as it is.

File: fonts/generate_font_metadata.py (hunk 3 above)
Comment (rettinghaus) [reply fonts/generate_font_metadata.py:48]: This file originally comes from https://github.com/w3c/smufl repo. We removed all the log file stuff and this one was overlooked. I'm still wondering if it should be in line with the upstream version and if we should push further fixes/improvements there?

File: setup.py (hunk 4 above)
Comment (ahankinson) [reply setup.py]: D203 is deprecated and should be disabled if your editor doesn’t do this already:

See:

https://opendev.org/openstack/keystoneauth/commit/cf520d9ca45dfbb09ca9bab90fa3d615bc950326

Comment (rettinghaus) [reply setup.py]: Yeah, I've seen that there has been a discussion. That's why I reverted the change already.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [unreleased]
+* Making Veroivo thread safe (@eNote-GmbH)
```
Comment (lpugin) [inline CHANGELOG.md:4]: Fix typo. Next time make a distinct PR with [skip-ci] for changelog updates. Thanks.

**Review by ahankinson — APPROVED** [review]

**Pull request merged**